velox/functions/prestosql/window/tests/RankTest.cpp

```diff
@@ -21,7 +21,28 @@ namespace facebook::velox::window::test {
 
 namespace {
 
-class RankTest : public WindowTestBase {};
+class RankTest : public WindowTestBase {
+ protected:
+  void testTwoColumnInput(const RowVectorPtr vectors) {
+    for (const auto& rankFunction : rankFunctions_) {
+      WindowTestBase::testTwoColumnInput({vectors}, rankFunction);
+    }
+  }
+
+  void testWindowFunction(
+      const std::vector<RowVectorPtr>& vectors,
+      const std::vector<std::string>& overClauses) {
+    for (const auto& rankFunction : rankFunctions_) {
+      WindowTestBase::testWindowFunction(vectors, rankFunction, overClauses);
+    }
+  }
+
+  const std::vector<std::string> rankFunctions_ = {
```

Comment:
Consider turning this test into parameterized test using function name as a parameter.

https://github.com/google/googletest/blob/main/docs/advanced.md#value-parameterized-tests

Reply:
Thanks for the suggestion, modified accordingly!